apps/omg_api/test/state/core_test.exs

```diff
@@ -629,6 +629,46 @@ defmodule OMG.API.State.CoreTest do
     end
   end
 
+  @tag fixtures: [:alice, :state_empty]
+  test "Output can have a zero value; can't be used as input though", %{alice: alice, state_empty: state} do
+    fee = %{eth() => 2}
+
+    state
+    |> Test.do_deposit(alice, %{amount: 10, currency: eth(), blknum: 1})
+    |> (&Core.exec(Test.create_recovered([{1, 0, 0, alice}], eth(), [{alice, 8}, {alice, 0}]), fee, &1)).()
+    |> success?
+    |> (&Core.exec(
+          Test.create_recovered([{2, 0, 0, alice}, {2, 1, 0, alice}], eth(), [{alice, 5}, {alice, 1}]),
```

Comment:
I think it should be `{2, 0, 1 ...` not `{2, 1, 0, ...}` 
...and including existing utxo `{2, 0, 0}` make it harder to reason about - please use just zero-output to show the behavior.

Reply:
Good catch. Fixed and simplified.